blocks/header-nav-chain-block/chains/header-nav-chain-block/default.jsx

```diff
@@ -27,29 +27,47 @@ import HorizontalLinksBar from './_children/horizontal-links/default';
 /* Global Constants */
 // Since these values are used to coordinate multiple components, I thought I'd make them variables
 // so we could just change the vars instead of multiple CSS values
-const navHeight = '56px';
+// const navHeight = '56px';
 const navZIdx = 9;
 const sectionZIdx = navZIdx - 1;
 
 /* Styled Components */
 const StyledNav = styled.nav`
   align-items: center;
-  position: relative;
+  width: 100%;
+  position: fixed;
+  top: 0px;
 
   .news-theme-navigation-bar {
     background-color: ${(props) => props.navBarBackground};
-    height: ${navHeight};
+    height: ${(props) => (props.scrolled ? props.shrinkHeight : props.navHeight)}px;
+    transition: 0.5s;
     z-index: ${navZIdx};
+
+    &.nav-logo, & > .nav-logo {
+      img {
+        height: auto;
+        max-height: ${(props) => (props.scrolled ? (props.shrinkHeight - 16) : (props.navHeight - 16))}px;
+        max-width: 240px;
```

Comment:
not sure why we have the value of max-with define at here?

Reply:
I'm not sure I understand - is the `max-width: 240px` style just not getting applied?

Reply:
it's working as expected, but I don't know why we have that value? where 240 px come from?

Reply:
Oh I'm actually not sure where that originated from since I didn't write that part. It looks like Jack had the last edit to that code so might be worth reaching out to him or Sara to find out. I'm not sure if it came from a design doc or something? In either case, it _might_ need to change since I'm noticing that the logo doesn't really expand to fit the available space **vertically** and it's probably because of the `max-width` limiting the amount that the logo can expand **horizontally**.